server/modules/search/elasticsearch/engine.js

```diff
@@ -182,7 +219,6 @@ module.exports = {
   async created(page) {
     await this.client.index({
       index: this.config.indexName,
-      type: '_doc',
```

Comment:
Does removing the `type` field in all mutation methods affect 6.x and 7.x?

Reply:
The docs indicate type was changed with 6.x and deprecated with 7.x to prepare for removal in 8.x. I had assumed it wasn't necessary with 6.x, but seems like it is still required in some situations.

Looks like more special cases are needed to continue 6.x support and add 8.x.